src/components/content/order/migrate/MigratingServices.tsx

```diff
@@ -0,0 +1,420 @@
+                    currentRegion={selectRegion}
+                    currentFlavor={selectFlavor}
+                />
+            ),
+            description: 'Select a destination for migrating the existing deployment.',
+        },
+        {
+            title: 'Deploy',
+            content: (
+                <ShowDeploy
+                    userAvailableServiceVoList={userAvailableServiceVoList}
+                    selectCsp={selectCsp}
+                    selectArea={selectArea}
+                    selectRegion={selectRegion}
+                    selectFlavor={selectFlavor}
+                    onFinish={handleFinish}
+                    deploying={isDeploying}
+                    formRef={formRef}
+                />
+            ),
+            description: 'Deploy the service on the new destination.',
+        },
+        {
+            title: 'Import Data',
+            content: <ImportServiceData />,
+            description: 'Import service data.',
+        },
+        {
+            title: 'Destroy',
+            content: (
+                <DestroyService
+                    userAvailableServiceVoList={userAvailableServiceVoList}
+                    selectCsp={selectCsp}
+                    selectArea={selectArea}
+                    selectRegion={selectRegion}
+                    selectFlavor={selectFlavor}
+                />
+            ),
+            description: 'Destroy the old deployment.',
+        },
+    ];
+
+    const items = steps.map((item) => ({ key: item.title, title: item.title, description: item.description }));
+
+    return (
+        <div className={'migrate-select-destination-class'}>
+            {tip}
+            <div className={'migrate-step-title-inner-class'}>
+                <Steps current={currentMigrationStep} items={items} />
+            </div>
+            <div className={'migrate-step-content-inner-class'}>{steps[currentMigrationStep].content}</div>
+            <div className={'migrate-step-button-inner-class'}>
+                <Space size={'large'}>
+                    {currentMigrationStep > MigrationSteps.ExportServiceData ? (
+                        <Button
+                            type='primary'
+                            className={'migrate-steps-operation-button-clas'}
+                            onClick={() => prev()}
+                            disabled={isDeploying || isDestroying}
+                        >
+                            Previous
+                        </Button>
+                    ) : (
+                        <></>
+                    )}
+
+                    {currentMigrationStep < MigrationSteps.DestroyTheOldService ? (
+                        <Button
+                            type='primary'
+                            className={'migrate-steps-operation-button-clas'}
+                            onClick={() => next()}
+                            disabled={isDeploying}
+                        >
+                            Next
```

Comment:
in case of "Deploy" step, the `next` button must only be enabled after the deployment is successful. 
As mentioned before, the next steps must blocking until the previous one is really completed.

Reply:
fixed.